packages/platform-sdk-ada/src/dto/transaction.ts

```diff
@@ -46,18 +51,29 @@ export class TransactionData extends DTO.AbstractTransactionData implements Cont
 		return this.data.outputs.map(
 			(output: Contracts.KeyValuePair) =>
 				new DTO.UnspentTransactionData({
-					amount: BigNumber.make(output.amount.quantity),
+					amount: BigNumber.make(output.value),
 					addresses: [output.address],
 				}),
 		);
 	}
 
 	public amount(): BigNumber {
```

Comment:
Could we use the `totalOutput` property for this? It seems to exclude the `fee` but I would need to check a transaction that contained changed to see if that is also taken into account.

Reply:
@faustbrian Change is also part of the output. That's why I had to do that ridiculous assumption (that change will be the last output address) and the the transferred amount are all the others.

Reply:
Will have a look at some other possible solutions then.